pkg/admission/clusterworkspacetypeexists/admission.go

```diff
@@ -41,7 +41,7 @@ import (
 	tenancyv1alpha1 "github.com/kcp-dev/kcp/pkg/apis/tenancy/v1alpha1"
 	"github.com/kcp-dev/kcp/pkg/authorization/delegated"
 	kcpinformers "github.com/kcp-dev/kcp/pkg/client/informers/externalversions"
-	tenancylisters "github.com/kcp-dev/kcp/pkg/client/listers/tenancy/v1alpha1"
+	tenancyv1alpha1lister "github.com/kcp-dev/kcp/pkg/client/listers/tenancy/v1alpha1"
```

Comment:
nit: this was the less common pattern and got squashed in the attempt to rewrite imports for consistency :)

Reply:
i couldn't remember / am planning on coming back to ensure consistency. thx for catching